app/views/handlers/submission.py

```diff
@@ -1,6 +1,7 @@
 from datetime import datetime
```

Comment:
- A test needs to be added to ensure that on submission the `submitted_at` property is indeed populated.
- Another test should to be added to ensure that the submitted time in the payload and questionnaire store matches.

Reply:
> test_store_and_get_submitted_at. does this
> as mentioned above there is no need to connect the 2

Reply:
Ya `test_submit_view_submitted_response_true_submitted_at_set` covers that it is populated. I missed that. But we should test that values are aligned now that they are.

Reply:
I reckon that's not needed, its create in one place and no transformation is done to it, so it's not going to be different. It's an argument you could use for any var across are app if used more than once

Reply:
> its create in one place and no transformation is done to it

How can we know without a test?. At the moment, it relies on cached property, if someone changed it to `propery` in future, it wouldn't necessarily match and no tests would break. Happy to let it go since it is not an official requirement.